---
title: Research in the Bagley Lab
layout: default
group: research
---

<div class="row">

# Our research
We are ecologists and evolutionary biologists and our research focuses on integrating genomics, ecological data, and computer modeling to reconstruct the evolutionary histories of species and populations. This work addresses basic questions (e.g. “Why are there so many species?”, "How do species adapt to their environment?") and applied questions (e.g. “How do we delimit and monitor species?”) about the evolution of freshwater and terrestrial organisms. Our research on evolutionary processes in biodiversity 'hotspots' takes us across the Americas, from the southeastern US to Central America and South America. 

Current research projects in the Bagley Lab, or Ecological and Evolutionary Genetics Laboratory (EEGL) facility at JSU, center on conservation genomics, phylogeography, phylogenomics, speciation/hybridization, and local adaptation genomics of North American and Neotropical plants (bellflowers, forest trees) and freshwater fishes. Projects are diverse, ranging from Andean bellflowers to central Brazilian fishes, and they are strongly focused on analyzing genomic data (e.g. ddRAD-seq, targeted sequence capture, and whole-genome sequence data) and species occurrences, although phenotypic data are also of interest. Questions about species, their relationships, and their past, present, and future responses (e.g. distributional shifts) to climate change and human impacts are especially of interest.

The Bagley Lab provides opportunities for students to engage in local and international research projects in ecological and evolutionary genetics employing cutting-edge techniques in high-throughput sequencing (grounded in bioinformatics) and ecological modeling. Research and internship opportunities for undergraduate students may focus on any of the following areas: 

-   applying techniques in ichthyology and genetic analysis to the assessment and conservation of native Alabama freshwater fish biodiversity; 
-   field- and genetics-based studies of native plant species across the southeastern U.S.; 
-   ecological niche modeling studies of species geographical distributions;
-   comparative analyses of plant or animal evolution and adaptation (e.g. gathering data from the field or literature for a group of related organisms with an available phylogeny, and then applying phylogenetic comparative methods to understand trait evolution in the group). 

Graduate student projects will depend on the interests of the student, and may also address the topics above, but they will more strongly emphasize applying molecular laboratory and computational approaches to key problems in molecular ecology and phylogenetics. Wherever possible, our laboratory will seek to develop interdisciplinary collaborations with other faculty in the Department of Biology and the Department of Chemistry and Geosciences.
  
  
<!--  
  The long-term goals of our research are to understand how protein conformational ensembles are reshaped by chemical, genetic, and physical perturbations. We seek to quantify how these perturbations impact protein function and organismal fitness. We are best known for creating multitemperature X-ray data collection approaches, which are especially powerful when paired with multiconformer computational modeling to reveal otherwise inaccessible features of conformational ensembles. Our group integrates high resolution structural biology (X-ray, EM, NMR), functional studies (deep mutational scanning, enzymology), and computation to study biological mechanisms and to improve protein engineering and small molecule discovery. Group members move fluidly between computation and experiment, often inventing new methods to answer their questions.

#### We work on many different systems and proteins as we find ourselves drawn, again and again, to the tension of high resolution data in structural biology: as resolution gets better, modeling becomes easier... until the resolution gets too good... and then the ensemble begins to reveal itself and the fun challenges begin.
-->
<br>
<br>

</div>

<!--
<div class="row">

### Identifying hidden alternative conformations of proteins and ligands in biophysical data

<div class="col-md-7 order-md-1">

We study proteins as conformational ensembles.
Although X-ray crystallography is an ensemble experiment, the results are typically summarized with a single static structure.
We develop [**software**](https://github.com/ExcitedStates/qfit-3.0) to discover the structural ensembles present in the crystal (or on the EM grid).
The ensemble nature of proteins highlighted by this work feeds into all of our mechanistic studies that interpret the functional effects of mutations, that characterize designed and artificially-evolved proteins, or that seek to modulate protein function with small molecules.
These methods development efforts are central to discovering new allosteric ligands, through high throughput crystallographic fragment screening efforts.
We are expanding, this direction to include modeling and validating protein structural data generated by cryo-electron microscopy (using EMRinger and ensemble modeling) and through integrative approaches to discover cryptic ligand binding sites.
These methods development efforts are central to discovering new allosteric ligands, through high throughput crystallographic fragment screening efforts.

</div>
<div class="col-md-5 order-md-2 align-self-center">
<img class="img-fluid" src="/static/img/pub/2017_biel.jpg" alt="qFit">
<!-- <a href="http://www.ucsf.edu"><img class="inline-block navb-icon" src="/static/img/ucsf_logo_white.svg" alt="University of California, San Francisco (UCSF) logo"></a> -->

</div>
</div>
<div class="row">

### Using multi-temperature X-ray data collection methods in mechanistic studies and ligand discovery campaigns

<div class="col-md-7 order-md-2">

We recognized that the standard practice of cryocooling crystals could distort protein conformations.
In both larger surveys and isolated mechanistic studies, we have demonstrated the value of room temperature data collection for revealing the structural basis of protein conformational dynamics, leading to new insights into enzymes, and increasing connections to dynamics studies from NMR and simulations.
Additionally, we have identified how temperature can bias small molecule discovery, leading some fragment sites inaccessible at cryogenic temperatures and repositioning critical water molecules.
</div>

<div class="col-md-5 order-md-1 align-self-center">
<img class="img-fluid" src="/static/img/pub/2018_keedy_hill.jpg" alt="ptp1b">
</div>
</div>
<div class="row">

### Determining structures that influence microbial-host interactions

<div class="col-md-7 order-md-1 ">

We are interested in the structural enzymology and potential therapeutic applications of the human enzymes that degrade chitin, the pervasive polymer that can cause inflammation in the context of allergy and asthma. Using high resolution cryo-electron microscopy (cryo-EM), we are studying the mechanisms of action to newly synthesized antibiotics (in collaboration with the [Seiple](https://seiplegroup.ucsf.edu/) and [Fujimori](https://fujimorilab.ucsf.edu) labs). With this platform, we are also pushing on cryo-EM data processing and refinement methods to enable structure-based drug design, now achieving resolutions better than 2.0 Å. With the improved molecules we generate, we are defining potential resistance mechanisms to these antibiotics using a combination of structural biology and deep mutational scanning experiments. In recent work, we are collaborating with the Bondy-Denomy lab to define the structural basis of anti-CRISPR proteins using a combination of X-ray crystallography, cryo-EM, and NMR. The mechanism of action for many of these proteins remains mysterious.
</div>

<div class="col-md-3 order-md-2 align-self-center">

<img class="img-fluid" src="/static/img/pub/2019_li_pellegrino.jpg" alt="antibiotic">
</div>
</div>
<div class="row">

### Identifying unifying concepts between systems and structural biology

<div class="col-md-7 order-md-2">


We are intrigued by the similarities in genetic epistasis and thermodynamic measurements and are applying these insights to large-scale studies of point mutants and posttranslational modifications. We attempt to connect the response to genetic (mutation), chemical (ligands), and physical (temperature) in experiments to define the basis of allosteric regulation and function. We like to bring an evolutionary biology perspective to the table to understand the relative importance of these different perturbations in shaping the proteins we see today.
</div>

<div class="col-md-5 order-md-1 align-self-center">
<img class="img-fluid" src="/static/img/pub/2020_gordon_jang_bouhaddou_xu_obernier_white_omeara_rezelj.png" alt="hairball">
</div>
</div>
<div class="row">

### Creating new time-resolved experiments to probe correlated motions in proteins

<div class="col-md-7 order-md-1">

A major limitation of most biophysical techniques is the inability to directly reveal correlations in motions between distinct regions of macromolecules.
We are taking advantage of the new capabilities of next-generation X-ray free electron laser (X-FEL) light sources to perform radiation damage-free imaging of proteins and to watch how protein ensembles respond when perturbed by rapid temperature jumps using the X-FEL.
At equilibrium, X-ray diffuse scattering has the potential to reveal these motions; however, we currently lack the ability to collect, integrate, and refine diffuse scattering data.
Our long-term goal is to increase the information content of every X-ray diffraction experiment to reveal atomic level coupling at high resolution and improved models of grouped flexibility at low resolution.
</div>

<div class="col-md-5 order-md-2 align-self-center ">
<img class="img-fluid" src="/static/img/pub/2019_thompson.jpg" alt="tjump">
</div>

</div>

-->
