---
title: Research in the Bagley Lab
layout: default
group: research
---

<div class="row">

# Our research
We are **ecologists and evolutionary biologists** and our research focuses on integrating genomics, ecological data, and computer modeling to reconstruct the evolutionary histories of species and populations. This work addresses basic questions (e.g. _“Why are there so many species?”_, _"How do species adapt to their environment?"_) and applied questions (e.g. _“How do we delimit and monitor species?”_) about the ecology and evolution of freshwater and terrestrial organisms. Our research on evolutionary processes in biodiversity 'hotspots' (e.g. defined [here](https://www.nature.com/articles/35002501)) takes us across the Americas, from the southeastern US to Central America and South America. 

Current research projects in the **"Bagley Lab"**, also known as the **Ecological and Evolutionary Genetics Laboratory (EEGL)** at [JSU](http://jsu.edu), center on conservation genomics, phylogeography, phylogenomics, speciation/hybridization, and local adaptation genomics of North American and Neotropical plants and freshwater fishes. Projects are diverse, ranging from Andean bellflowers to central Brazilian fishes, and they are strongly focused on analyzing genomic data (e.g. ddRAD-seq, targeted sequence capture, and whole-genome sequence data) and species occurrences, although phenotypic data are also of interest. Questions about species, their relationships, and their past, present, and future responses (e.g. distributional shifts) to climate change and human impacts are especially of interest.
<br>
<br>

</div>

<div class="row">

### Comparative phylogeography and phylogeographic meta-analyses

We study phylogeography (evolutionary and geographic histories of genetic lineages within and among closely related species) at the single-species and comparative levels.
In this field, we are working on novel empirical studies of freshwater fish phylogeography in North America and Brazil based on high-throughput sequence data (e.g. ddRAD-seq, DArT-seq), as well as synthetic comparative reviews of published phylogeographical literature in the context of regional ecological and geological settings. 
We also have a new ongoing project in collaboration with [Fernando Alda's lab](http://fernandoalda.weebly.com) at the University of Tennessee-Chattanooga that is focused on combining georeferenced legacy phylogeographic data (DNA sequence data, esp. mtDNA, available from online repositories) with environmental data to conduct regional and global meta-analyses of fish phylogeography. This exciting project is made possible through our collaboration on [PhylogatR](https://phylogatr.org), a project funded through NSF grants to [Bryan Carstens](https://carstenslab.osu.edu) and [Tara Pelletier](https://sites.google.com/site/taraapelletier/).

<p align="center">
	<a href="https://phylogatr.org">
	<img class="img-fluid" src="/static/img/PhylogatR_logo.png" alt="PhylogatR" width="450"></a>
</p>

</div>

<div class="row">

### Conservation genomics

Our expertise in population genomics and molecular and bioinformatics approaches for analyzing next-generation sequencing data places us in an excellent position to apply these cost-effective and informative tools to applied problems in conservation and management. 
We are initiating a project on conservation genomics of Alabama's threatened and endangered minnow species, which was recently approved for funding from the Alabama Division of Wildlife and Freshwater Fisheries (DWFF). 
This project has two parts, one for each species. The species are the Blue Shiner (_Cyprinella caerulea_), which is restricted to the Coosa River system of the Mobile Basin, and the Cahaba Shiner (_Notropis cahabae_) native to the Cahaba River basin. Blue Shiner is pictured below.

<p align="center">
	<a href="https://www.outdooralabama.com/shiners/blue">
	<img class="img-fluid" src="/static/img/Shinerblue400A.jpg" alt="Blue Shiner" width="450"></a>
</p>

</div>


