---
title: Research in the Bagley Lab
layout: default
group: research
---

<div class="row">

# Our research
We are ecologists and evolutionary biologists and our research focuses on integrating genomics, ecological data, and computer modeling to reconstruct the evolutionary histories of species and populations. This work addresses basic questions (e.g. “Why are there so many species?”, "How do species adapt to their environment?") and applied questions (e.g. “How do we delimit and monitor species?”) about the evolution of freshwater and terrestrial organisms. Our research on evolutionary processes in biodiversity 'hotspots' takes us across the Americas, from the southeastern US to Central America and South America. 

Current research projects in the Bagley Lab, or Ecological and Evolutionary Genetics Laboratory (EEGL) facility at JSU, center on conservation genomics, phylogeography, phylogenomics, speciation/hybridization, and local adaptation genomics of North American and Neotropical plants (bellflowers, forest trees) and freshwater fishes. Projects are diverse, ranging from Andean bellflowers to central Brazilian fishes, and they are strongly focused on analyzing genomic data (e.g. ddRAD-seq, targeted sequence capture, and whole-genome sequence data) and species occurrences, although phenotypic data are also of interest. Questions about species, their relationships, and their past, present, and future responses (e.g. distributional shifts) to climate change and human impacts are especially of interest.

The Bagley Lab provides opportunities for students to engage in local and international research projects in ecological and evolutionary genetics employing cutting-edge techniques in high-throughput sequencing (grounded in bioinformatics) and ecological modeling. Research and internship opportunities for undergraduate students may focus on any of the following areas: 

-   applying techniques in ichthyology and genetic analysis to the assessment and conservation of native Alabama freshwater fish biodiversity; 
-   field- and genetics-based studies of native plant species across the southeastern U.S.; 
-   ecological niche modeling studies of species geographical distributions;
-   comparative analyses of plant or animal evolution and adaptation (e.g. gathering data from the field or literature for a group of related organisms with an available phylogeny, and then applying phylogenetic comparative methods to understand trait evolution in the group). 

Graduate student projects will depend on the interests of the student, and may also address the topics above, but they will more strongly emphasize applying molecular laboratory and computational approaches to key problems in molecular ecology and phylogenetics. Wherever possible, our laboratory will seek to develop interdisciplinary collaborations with other faculty in the Department of Biology and the Department of Chemistry and Geosciences.
  
  
<!--  
  The long-term goals of our research are to understand how protein conformational ensembles are reshaped by chemical, genetic, and physical perturbations. We seek to quantify how these perturbations impact protein function and organismal fitness. We are best known for creating multitemperature X-ray data collection approaches, which are especially powerful when paired with multiconformer computational modeling to reveal otherwise inaccessible features of conformational ensembles. Our group integrates high resolution structural biology (X-ray, EM, NMR), functional studies (deep mutational scanning, enzymology), and computation to study biological mechanisms and to improve protein engineering and small molecule discovery. Group members move fluidly between computation and experiment, often inventing new methods to answer their questions.

#### We work on many different systems and proteins as we find ourselves drawn, again and again, to the tension of high resolution data in structural biology: as resolution gets better, modeling becomes easier... until the resolution gets too good... and then the ensemble begins to reveal itself and the fun challenges begin.
-->
<br>
<br>

</div>


