---
title: Research in the Bagley Lab
layout: default
group: research
---

<div class="row">

# Our research
We are **ecologists and evolutionary biologists** and our research focuses on integrating genomics, ecological/field data, and computer modeling to address problems in molecular ecology, 
distributional ecology, conservation, and biodiversity science. Our research answers basic and applied questions about the ecology and evolution of freshwater and terrestrial organisms and 
can broadly be divided into **two categories: (1) Evolutionary Processes and (2) Applied Ecological Research.** 
<!-- histories of species and populations. This work addresses basic questions (e.g. _“Why are there so many species?”_, _"How do species adapt to their environment?"_) and 
applied questions (e.g. _“How do we delimit and monitor species?”_) about the ecology and evolution of freshwater and terrestrial organisms. -->
Our research on evolutionary 
processes focuses on biodiversity 'hotspots' (e.g. defined [here](https://www.nature.com/articles/35002501)) and takes us across the Americas, from the southeastern US to Central 
America and South America. Our applied research focuses on two sub-areas: (i) ecological niche modeling (ENM) of ecological requirements (niches) to predict species past, present, and 
future distributions, including species responses (e.g. distributional shifts) to climate change and human impacts, and (ii) biological assessment (e.g. biomonitoring) and community 
ecology of the diverse fish assemblages of Alabama and the southeastern US.

Current research projects in the **"Bagley Lab"** of ecology and evolution at [JSU](http://jsu.edu) center on 
<!--, also known as the Ecological and Evolutionary Genetics Laboratory (EEGL) at [JSU](http://jsu.edu), center on -->
ecological niche modeling, conservation genomics, phylogeography, phylogenomics, speciation/hybridization, and local adaptation genomics of North American and Neotropical plants and freshwater 
fishes and are described in more detail below. While our main area of organismal expertise is ichthyology and most projects focus on fishes, projects are diverse, ranging from Andean bellflowers to central Brazilian fishes. 
<!-- , and they are strongly focused on analyzing genomic data (e.g. ddRAD-seq, 
targeted sequence capture, and whole-genome sequence data) and species occurrences, although phenotypic data are also of interest. Questions about species, their 
relationships, and their past, present, and future responses (e.g. distributional shifts) to climate change and human impacts are especially of interest. -->
<br>
<br>

</div>

<div class="row">
# 1. Evolutionary processes

### Comparative phylogeography and phylogeographic meta-analyses

We study <a href="https://justinbagley.org/pages/phylogeog.html">phylogeography</a> (evolutionary and geographic histories of genetic lineages within and among closely related species) at the single-species and comparative levels.
In this field, we are working on novel empirical studies of freshwater fish phylogeography in North America and Brazil based on high-throughput sequence data (e.g. 
ddRAD-seq, DArT-seq), as well as synthetic comparative reviews of published phylogeographical literature in the context of regional ecological and geological settings. 
We also have a new ongoing project in collaboration with [Fernando Alda's lab](http://fernandoalda.weebly.com) at the University of Tennessee-Chattanooga that is focused 
on combining georeferenced legacy phylogeographic data (DNA sequence data, esp. mtDNA, available from online repositories) with environmental data to conduct regional and 
global meta-analyses of fish phylogeography. This exciting project is made possible through our collaboration on [PhylogatR](https://phylogatr.org), a project funded 
through an NSF grant to [Bryan Carstens](https://carstenslab.osu.edu) and [Tara Pelletier](https://sites.google.com/site/taraapelletier/).

<p align="center">
	<a href="https://phylogatr.org">
	<img class="img-fluid" src="/static/img/PhylogatR_logo.png" alt="PhylogatR" width="450"></a>
</p>

</div>

<div class="row">

<!-- #### Single-species phylogeography -->
<!-- # aspen_popstruct_fig1.png -->
<!--
<p align="center">
	<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ece3.6214"> # BETTER URL (link to full text):  http://bagleylab.com/static/pdf/2020_bagley_heming_gutierrez_et_al.pdf
	<img class="img-fluid" src="/static/img/aspen_popstruct_fig1.png" alt="GBS phylogeography of quaking aspen (Populus tremuloides; Bagley et al. 2020)." width="450"></a>
</p>

-->

<div class="row">

### Single-species phylogeography and demography

Whereas comparative phylogeography (mentioned _above_) can tell us about broad-scale patterns of evolution of species in a region, comparative studies rely on scaling up from many studies of individual species. These single-species phylogeography studies each tell us about the spatial-demographic histories of a different species or group (including metazoa and viruses). We have been working on phylogeography of terrestrial and aquatic ecosystems of temperate and Neotropical areas for over 15 years. While Dr. Bagley's dissertation work focused on phylogeography and species delimitation in Central American freshwater fishes, our work looking at the histories of individual species has focused more recently on individual forest trees such as quaking aspen (_Populus tremuloides_) and pines (genus _Pinus_), and now that we are located back in Alabama our lab focus is shifting primarily to using phylogeography approaches combined with ecological niche modeling to understand evolutionary processes in species from the North American southeast (a known aquatic biodiversity 'hotspot'). We also have quite a bit of data on the phylogeography of fishes from the Brazilian Cerrado 'hotspot' that we are working to publish.

<p align="center">
	<a href="http://bagleylab.com/static/pdf/2020_bagley_heming_gutierrez_et_al.pdf">
	<img class="img-fluid" src="/static/img/aspen_fig1_pop_struct.png" alt="Quaking aspen phylogeography and population structure (Bagley et al. 2020)." width="450"></a>
</p>

</div>


<!-- ### Conservation genomics

Our expertise in population genomics and molecular and bioinformatics approaches for analyzing next-generation sequencing data places us in an excellent position to 
apply these cost-effective and informative tools to applied problems in conservation and management. 
We are initiating a project on conservation genomics of Alabama's threatened and endangered minnow species, which was recently approved for funding from the Alabama 
Division of Wildlife and Freshwater Fisheries (DWFF). 
This project has two parts, one for each species. The species are the <a href="https://www.outdooralabama.com/shiners/blue">Blue Shiner (_Cyprinella caerulea_)</a>, which 
is restricted to the Coosa River system of the 
Mobile Basin, and the <a href="https://www.outdooralabama.com/shiners/cahaba">Cahaba Shiner (_Notropis cahabae_)</a> native to the Cahaba River basin. Blue Shiner is 
pictured below (_Photo courtesy of Dr. Pat O'Neil, GSA_).

<p align="center">
	<a href="https://www.outdooralabama.com/shiners/blue">
	<img class="img-fluid" src="/static/img/Shinerblue400A.jpg" alt="Blue Shiner" width="450"></a>
</p>

</div>
-->

<div class="row">

### Phylogenomics & species delimitation

Lab research in systematics and taxonomy primarily focuses on discovering and determining evolutionary relationships among species, and clarifying species 
limits, through analyses of DNA sequence data and morphological characters. Hypotheses of phylogenetic relationships are reconstructed based on broad taxon and character 
sampling using maximum-likelihood and Bayesian inference analyses, relaxed molecular clocks, clock-partitioning schemes, and tip- and fossil-calibrated models (e.g. 
fossilized birth-death process). Our work also seeks to describe new biodiversity to improve biodiversity accounting. This work focuses on understanding diversification 
and the evolution of adaptive traits (e.g. reproductive mode) in monophyletic clades of temperate and Neotropical plants (bellflowers, pines), freshwater fishes, and 
squamate reptiles. Recent <a href="http://bagleylab.com/publications/">papers</a> focused on

-   Describing two new species of suckermouth armored catfishes (Loricariidae) from central Brazil
-   Phylogenomics of recent, rapid angiosperm radiations with Andean-centered distributions, using _Burmeistera_ (Campanulaceae) as a case study
-   Tailless leaf-nosed _Anoura_ bats that pollinate _Burmeistera_ bellflowers
-   Integrating molecules, morphology, and fossils to infer relationships among North American 'sucker' fishes (Catostomidae)

<p align="center">
	<a href="http://bagleylab.com/static/pdf/2020_bagley_uribeconvers_carlsen_muchhala.pdf">
	<img class="img-fluid" src="/static/img/burmeistera_PI.png" alt="Burmeistera time tree and phylogenetic informativeness (Bagley et al. 2020)." width="450"></a>
</p>

</div>


<!--
><div class="row">

### Phylogeography of temperate and Neotropical plants and animals

We have been working on phylogeography of temperate and Neotropical freshwater fishes (Brazilian Cerrado and Mesoamerica hotspots, plus North American southeast) and plants (e.g. forest trees) for the last 15 years.

<a href="https://justinbagley.org/pages/phylogeog.html">Phylogeography</a>, the study of the geographical distributions of genetic lineages within and among closely 
related species, has assumed a central role in molecular ecology 
for inferring the historical and ecological processes influencing present-day biodiversity patterns. Along with collaborators from the U.S., Brazil, and Mexico, Dr. Bagley has 
been involved in a number of projects using single-species and comparative (multispecies) phylogeography approaches to improve our understanding of the diversification, 
stability, and assembly of North American and Neotropical biotic communities over the last 15 years. 

This work currently focuses on terrestrial and aquatic ecosystems of three intriguing systems that present different contexts for 
understanding evolutionary processes: 1) North American southeast (an aquatic biodiversity hotspot) and the 2) Mesoamerica (Central America) and 3) Brazilian Cerrado 
biodiversity 'hotspots'.

<p align="center">
	<a href="https://www.outdooralabama.com/shiners/blue">
	<img class="img-fluid" src="/static/img/Shinerblue400A.jpg" alt="Blue Shiner" width="450"></a>
</p>

</div>

-->

<div class="row">

### Genomics of speciation & local adaptation, with implications for climate change

Understanding how barriers to gene flow arise, and new species boundaries are formed and maintained, is a central goal of evolutionary biology. Yet, the study of 'speciation', presents numerous challenges and requires integrative perspectives taking into account geographical factors (biogeography) and range dynamics, niche evolution, reproductive isolation, as well as the evolution of neutral and adaptive genetic variation in the context of demographic history. We have become interested for several years now in the ways in which plant versus animal systems can reveal different aspects of the speciation process.

Our recent and on-going projects on southwestern white pine (_Pinus strobiformis_) highlight the role of hybridization and extrinsic factors during ecological speciation. We are also using genome scan and environmental association analyses to understand local adaptation to climatic gradients and challenging environments (high elevation) in this system, with important implications for understanding tree responses to climate change. By contrast, some Neotropical fish lineages that Dr. Bagley has studied (e.g. livebearers; Poeciliidae) are more predisposed to speciation through intrinsic pre- or postmating isolation, and exhibit presumed niche conservatism but a high degree of phenotypic variation. We are eager to develop future projects expanding our speciation and adaptation work along these two avenues of research.

<p align="center">
	<a href="http://bagleylab.com/static/pdf/2018_menon_bagley_friedline_whipple_et_al.pdf">
	<img class="img-fluid" src="/static/img/swwp_figb_pop_struct.png" alt="Southwestern white pine hybrid zone and genetic structure (Menon et al. 2018)." width="450"></a>
</p>

</div>



<div class="row">

# 2. Applied ecological research
Organismally, Dr. Bagley is a professionally trained ichthyologist and he has been studying and publishing on freshwater fishes since 2006 (15 years of experience).

Thus, while the lab has worked on a variety of organisms and many 'basic'-science questions (projects above), we are also fundamentally interested in conducting applied research into questions of relevance to the biological assessment (e.g. biomonitoring, fisheries research), community ecology, and conservation of the diverse fish assemblages of Alabama and the southeastern US. We are working with state agencies to develop projects in this applied arena. Currently, we are most interested in 

-   Applying our expertise in phylogeography and genomics to conservation issues, e.g. see Conservation genomics project(s) below
-   Development of wadeable and non-wadeable Index of Biotic Integrity (IBI) approaches for assessing state/regional waters
-   Environmental niche modeling (ENM) to predict species past, present, and future distributions in response to climate change
-   Environmental DNA (eDNA) metabarcoding of freshwater rivers and streams of Alabama and surrounding areas (e.g. Mobile basin extensions into Mississippi and Georgia)

### Conservation genomics

Our expertise in population genomics and molecular and bioinformatics approaches for analyzing next-generation sequencing data places us in an excellent position to 
apply these cost-effective and informative tools to applied problems in conservation and management. 
We are initiating a project on conservation genomics of Alabama's threatened and endangered minnow species, which was recently approved for funding from the Alabama 
Division of Wildlife and Freshwater Fisheries (DWFF). 
This project has two parts, one for each species. The species are the <a href="https://www.outdooralabama.com/shiners/blue">Blue Shiner (_Cyprinella caerulea_)</a>, which 
is restricted to the Coosa River system of the 
Mobile Basin, and the <a href="https://www.outdooralabama.com/shiners/cahaba">Cahaba Shiner (_Notropis cahabae_)</a> native to the Cahaba River basin. Blue Shiner is 
pictured below (_Photo courtesy of Dr. Pat O'Neil, GSA_).

<p align="center">
	<a href="https://www.outdooralabama.com/shiners/blue">
	<img class="img-fluid" src="/static/img/Shinerblue400A.jpg" alt="Blue Shiner" width="450"></a>
</p>

</div>


<br>
<br>

</div>


<div class="row">

### Ecological niche modeling (ENM) of species responses to climate change

We are interested in applications of niche theory to questions about the actual and potential distributions of species in the past, present, and future, through [ecological niche modeling (ENM)](https://www.youtube.com/watch?v=NGGOfHKISZY) 
(also see [this book](https://press.princeton.edu/books/paperback/9780691136882/ecological-niches-and-geographic-distributions-mpb-49)). 
The ENM approach has many applications, and those that we find most exciting include its uses for predicting species responses to global climate change (GCC) and biodiversity assessment/accounting. 
Currently, Dr. Bagley has an ongoing project with students in his Ecology CURE course and in the Bagley Lab to predict the responses of Alabama's coastal dune plant species to GCC. We are reconstructing 
the niches and predicting present and future distributions (under climate and carbon emissions scenarios for year 2070; ~50 years from now) for 14 endemic foredune and backdune species, including charismatic 
species such as sea oats (_Uniola paniculata_) as well as taxa that are lesser-known to the general public, such as sea purslane (_Sesuvium portulacastrum_) and Florida rosemary (_Ceratiola ericoides_). 
We are grateful to the JSU Department of Biology and College of Science and Mathematics for funding and support of this project.

<p align="center">
	<!--<a href="https://www.outdooralabama.com/shiners/blue"> -->
	<img class="img-fluid" src="/static/img/ENM_Spumilus_Spat_3panel_pic_curr_fut.png" alt="Blue Shiner" width="850"><!--</a>-->
</p>

</div>
